src/care/ProfilePlugin.h

```diff
@@ -0,0 +1,31 @@
+#ifndef _CARE_ProfilePlugin_H_
+#define _CARE_ProfilePlugin_H_
+
+#include "RAJA/util/PluginStrategy.hpp"
+
+namespace care{
+
+	class ProfilePlugin : public RAJA::util::PluginStrategy
```

Comment:
Could you try the instructions in this article to fix the formatting of DebugPlugin.h and ProfilePlugin.h using vim? https://vim.fandom.com/wiki/Fix_indentation

A prerequisite is that you modified the .vimrc file to use 3 spaces per tab and convert tabs to spaces. I believe I gave you the instructions on how to do that, but let me know if you need them again.